packages/pin-input/src/use-pin-input.ts

```diff
@@ -86,13 +79,26 @@ export interface UsePinInputProps {
    * If `true`, the pin input component is put in the invalid state
    */
   isInvalid?: boolean
+  /**
+   * The type of values the pin-input should allow
+   */
+  type?: "alphanumeric" | "number"
+  /**
+   * If `true`, the input's value will be masked just like `type=password`
+   */
+  mask?: boolean
 }
 
 function toArray(value?: string) {
-  if (typeof value === "string") {
-    return value.split("")
-  }
-  return undefined
+  return typeof value === "string" ? value.split("") : undefined
```

Comment:
Would we actually need to check type here anymore if `value` is typed to `string`?

Reply:
It's because the value could be `undefined` as well. I could have just done `value?.split("")` haha. Will refactor